packages/terra-list/src/SelectableUtils.js

```diff
@@ -118,6 +124,9 @@ const wrappedOnKeyDownForItem = (item, index, handleOnChange) => {
   };
 };
 
+/**
+ * Returns an objet containing accessiblity and selectable projects.
```

Comment:
selectable ~projects~. properties?

Reply:
good catch, i think the bbq is clouding my judgement

Reply:
Hey, its clouding mine too